camel/datahubs/models.py

```diff
@@ -19,4 +19,6 @@
 class Record(BaseModel):
     id: Optional[str] = None
     metadata: Optional[Dict[str, Any]] = None
-    content: Dict[str, Any]
+    
+    class Config:
+        extra = "allow"  # Allow any additional fields
```

Comment:
for example：
# Create a basic record
record = Record(id="001", metadata={"source": "web"})

# Add extra fields (this is allowed since extra="allow")
record = Record(
id="001",
metadata={"source": "web"},
content="some text", # extra fields
timestamp="2024-01-01" # Additional fields
)


![image](https://github.com/user-attachments/assets/a94e4ab4-eb16-4245-8352-d5bba8bf4fe7)
![image](https://github.com/user-attachments/assets/bec87538-8d63-48d6-9517-bea8b729514c)

Reply:
could we set `content` as optional instead of removing this parameter? Otherwise user may feel confused where to add their core content information

Reply:
sounds good, already done it